CPT331.Data.Parsers/VicXmlParser.cs

```diff
@@ -0,0 +1,71 @@
+﻿#region Using References
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+
+using CPT331.Core.ObjectModel;
+
+#endregion
+
+namespace CPT331.Data.Parsers
+{
+	public class VicXmlParser
+	{
+		//	Number of offences by geographic area and offence type, year ending December 2014									
+		public VicXmlParser(string filename)
```

Comment:
This type is very similar to the NSW parser, and will be refactored in time to push common implementation into a base implementation.